front/lib/models/assistant/conversation.ts

```diff
@@ -492,7 +505,7 @@ Message.belongsTo(UserMessage, {
 });
 
 AgentMessage.hasOne(Message, {
-  as: "agentMessage",
+  as: "message",
```

Comment:
This looks extremely off that we have a "as" that is not aligned with the foreignKey name?

Also we want this to be algned across AgentMessage/UserMessage for sure. You can't just change one and not the other.

Reply:
Always value consistency first. You can't change one thing and not the other here.

Reply:
Reverting it for speed